lib/builder/builder.js

```diff
@@ -622,6 +623,15 @@ export default class Builder {
     this.customFilesWatcher = chokidar
       .watch(customPatterns, options)
       .on('change', refreshFiles)
+
+    // Watch for nuxt.config.js and user-defined changes
+    const nuxtConfigPatterns = _.concat(
```

Comment:
I don't think that `nuxtConfigPatterns` is the right name, maybe something like `nuxtRestartWatch`